_posts/2022-04-28-introducing-uswds-3-0.md

```diff
@@ -36,7 +36,7 @@ USWDS 3.0 stylesheets use the most modern version of the Sass module syntax. The
 
 Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
 
-[Read more about USWDS Packages]({{ site.baseurl }}/components/packages/) →
+[Read more about USWDS packages]({{ site.baseurl }}/components/packages/) →
 
 ### Simplify your project settings
 
```

Comment:
@thisisdano Feeling like we need to update the [Settings page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/develop/documentation/settings/) for 3.0, and linking to it in this section might cause confusion. For today, would it make sense to reroute the link to the [Sass modules section of the migration guide ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/develop/documentation/migration/#5-update-to-sass-module-syntax)?

Reply:
Made updates to settings page here: https://github.com/uswds/uswds-site/pull/1564